tessen

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# vim: set expandtab ts=2 sw=2 sts=4:
```

Comment:
Please move this after the line which mentions the copyright details.

Reply:
Also, I'm not sure but should the value of `sts` be 4 or 2? [I'm using](https://github.com/ayushnix/dotfiles/blob/28fcaa1fe074b9388e7dcfd6b3eb1706288717f5/private_dot_config/nvim/ftplugin/sh.vim#L9) 2.